src/services/session/test/controllers/SessionSelectorControllerSpec.js

```diff
@@ -1,281 +1,270 @@
-/*global define,describe,beforeEach,jasmine,Promise,it,expect,afterEach,
-    waitsFor,runs*/
+// import SessionSelectorController from '../SessionSelectorController';
+
+// this would be better tested in an e2e test
```

Comment:
do these not work? it would be good to have some tests until we get e2e tests

Reply:
these are Vue component tests, so I think e2e is the better way to go.

Reply:
I agree. But we don't have e2e tests yet. I think we should at least create an issue for these if we can't fix them now.